lms/static/scripts/frontend_apps/components/URLPicker.tsx

```diff
@@ -72,37 +67,14 @@ export default function URLPicker({
     >
       <div className="space-y-4">
         <p>Enter the URL of any publicly available web page or PDF:</p>
```

Comment:
I'm wondering if it would make sense to display this text with the same value we set as `aria-label`. In that case we could have a `label` or `title` prop instead, that is used for both values, and move this `p` inside `URLPickerForm`.